apps/site/lib/site_web/controllers/schedule/line/helpers.ex

```diff
@@ -80,10 +80,36 @@ defmodule SiteWeb.ScheduleController.Line.Helpers do
   defp do_get_branch_route_stops(route, direction_id, route_pattern_id) do
     route.id
     |> get_route_patterns(direction_id, route_pattern_id)
-    |> Enum.filter(&by_typicality(&1, route_pattern_id))
+    |> filter_by_min_typicality()
+    # The below row is for testing purposes for the ferry ONLY
+    # This is needed until a specific routepattern has a distinct typicality
+    |> temporary_hh_ferry_only_func(route.id)
     |> Enum.map(&stops_for_route_pattern/1)
   end
 
+  @spec filter_by_min_typicality([RoutePattern.t()]) :: [RoutePattern.t()]
+  defp filter_by_min_typicality(route_patterns) do
+    route_patterns
+    |> Enum.reduce({nil, []}, fn
+      %RoutePattern{typicality: typicality} = r, {t, a} when typicality < t -> {typicality, [r]}
+      %RoutePattern{typicality: typicality} = r, {typicality, a} -> {typicality, [r | a]}
+      _, acc -> acc
+    end)
+    |> elem(1)
+    |> Enum.reverse()
+  end
+
+  @spec filter_by_min_typicality([RoutePattern.t()]) :: [RoutePattern.t()]
+  defp temporary_hh_ferry_only_func(route_patterns, id) do
+    if id == "Boat-F1" do
+      route_patterns
+      |> hd()
+      |> (fn a -> [a] end).()
```

Comment:
The output of hd() isn't an enumerable - it's just a single stop.  Is there a simple way to toss it into an array?

Reply:
Dang I was thinking the `RoutePattern%{}` struct would be an enumerable. In that case I'm not sure I can think of a way that's any simpler than what you have already, but something like this might read a bit clearer? 🤷‍♀️ 
```Elixir
[first_route_pattern | _ ] = route_patterns
[first_route_pattern]
```

Reply:
In the future, `List.wrap` does what you want:
```elixir
iex> List.wrap(5)
[5]
iex> List.wrap(nil)
[]
```